Nlpcraft 339 - Added test for PhoneModel example

rahul3 opened the pull request:
Added test for PhoneModel example, per JIRA [NLPCRAFT-339](https://issues.apache.org/jira/projects/NLPCRAFT/issues/NLPCRAFT-339)  

To test: `bin/nlpcraft.sh test-model --cp=<your-path>/incubator-nlpcraft/nlpcraft-examples/phone/target/classes --mdls=org.apache.nlpcraft.examples.phone.PhoneModel  
`

skhdl commented:
hi!
I have few comments
1) Look at weather_example pom.xml. 
Similar should be for `phone` example pom.xml
 - junit should be added to pom.xml
 - test should be excluded on verify phase.
2) Short memo should be added for this test
 - change config for server  (tokenProviders = “nlpcraft,google”)
 - set GOOGLE_APPLICATION_CREDENTIALS
3) this test is special case. We have to disable it by default (Look at junit `disable` annotations). 
This tests should be started manually only (developer should enable it and follow the memo)

rahul3 commented:
Hello,

Thank you for your comments. Your help is much appreciated. Your comments are addressed below in the same order:

1. I have updated the `pom.xml`. The commit hash is `382c371`.If you could please take a look and let me know if you see any issues, I would appreciate it.

2. I've added the memo in the `PhoneModel.java` and the `NCModelValidationSpec.scala` file for now.

3. I have disabled the test. Commit hash `7a763d2` . Let me know if you see any issues.
